### PR TITLE
Add new case styling

### PR DESCRIPTION
**Pull request by njons**
Styling of the first page after clicking ADD NEW in the stations grid

### CODE REVIEW

File: src/components/Cases/style.scss [hunk 1]
```diff
@@ -1,36 +1,36 @@
-#banner-wrapper {
-  // background-color: grey;
-  display: flex;
-  flex-direction: column;
-}
+// #banner-wrapper {
+//   // background-color: grey;
+//   display: flex;
```
Comment (martingaston) [inline src/components/Cases/style.scss]: I think if this is just a commented out file now it's probably best to remove it entirely. Don't need to keep it around.

Comment (njons) [reply src/components/Cases/style.scss]: will do!

Comment (martingaston) [reply src/components/Cases/style.scss]: Rest looks great tho :+1:

Comment (njons) [reply src/components/Cases/style.scss]: I realise that the Case page still needs this .scss until we have changed those files to styled components!! so I have uncommented it so it renders properly for now!

Comment (njons) [reply src/components/Cases/style.scss]: cool will remove it, some styles won't render in the Revision flow but Eve is working on that so they will work once we merge her work

**Pull request merged**